### PR TITLE
Misc changes

### PR DESCRIPTION
**Pull request by vincentneko**
Fix Solo game broken if you win before last turn with Venus Next #355
Fix Cannot use Celestic action on first turn #351
Fix Only four colonies in a two player game #349
Suggestion: Expandable logging #288, thanks @alrusdi for this one
Enable Ants and Predators for solo game play

### CODE REVIEW

File: src/Player.ts [hunk 1]
```diff
@@ -596,6 +596,15 @@ export class Player {
     }
 
     public worldGovernmentTerraforming(game: Game) {
+
+      // Test if this is needed, usefull for solo play
+      if (game.getTemperature() >= constants.MAX_TEMPERATURE 
+        && game.getOxygenLevel() >= constants.MAX_OXYGEN_LEVEL
+        && game.board.getOceansOnBoard() >= constants.MAX_OCEAN_TILES
+        && game.getVenusScaleLevel() >= constants.MAX_VENUS_SCALE) {
```
Comment (bafolts) [inline src/Player.ts]: If you are not playing the venus game this will always be false.

File: src/Player.ts [hunk 2]
```diff
@@ -1446,7 +1455,7 @@ export class Player {
         game.getGeneration() === 1 &&
             this.corporationCard !== undefined &&
             this.corporationCard.initialAction !== undefined &&
-            !this.actionsThisGeneration.has(this.corporationCard.name) &&
+            !this.actionsThisGeneration.has("CORPORATION_INITIAL_ACTION") &&
```
Comment (bafolts) [inline src/Player.ts]: What was the issue with using `this.corporatinCard.name`?

Comment (vincentneko) [reply src/Player.ts]: It was consumming the corporation action.

File: src/Player.ts (hunk 1 above)
Comment (vincentneko) [reply src/Player.ts]: World gov mechanism is tight with venus

**Pull request merged**